app/views/videos/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<article class="h-entry" id="video-<%= @video.id %>">
+  <%= render "video", video: @video, linked: false %>
```

Comment:
Same as above.

`render video` will probably get you what you want after changing the `link_to` in the partial.